agriculture_shop/__manifest__.py

```diff
@@ -37,6 +37,7 @@
         'data/website_view.xml',
         'data/website_theme_apply.xml',
         'data/ir_model_data.xml',
+        'data/konwledge_tour.xml',
```

Comment:
typo to correct in all modules
```suggestion
        'data/knowledge_tour.xml',
```